### ISSUE TITLE
Python and R data downloaders do not work out of the box

### ISSUE THREAD

**Issue by jeffcsauer** [stanford-policylab/opp#82]
Hi all, 

Thank you so much for striving to make this data and analysis as open and reproducible as possible! 

Quick question. I am having trouble getting either the python cli or R package to work out of the box. Any thoughts on what might be going on? 

When trying with python, I navigate to a fresh folder, git clone the repo, and `cd opp`. However, when I try any of the download commands I run into the following errors:

```cmd
(geo) C:\Users\jeffe\Dropbox\SpatialPolice\1_Data\opp>./download.py -t shapefiles
Traceback (most recent call last):
  File "C:\Users\jeffe\Dropbox\SpatialPolice\1_Data\opp\py\download.py", line 149, in <module>
    download(
  File "C:\Users\jeffe\Dropbox\SpatialPolice\1_Data\opp\py\download.py", line 20, in download
    d = get_urls()
  File "C:\Users\jeffe\Dropbox\SpatialPolice\1_Data\opp\py\download.py", line 62, in get_urls
    state, city, file_type = info(url)
  File "C:\Users\jeffe\Dropbox\SpatialPolice\1_Data\opp\py\download.py", line 83, in info
    state, city = resolve_location(url)
ValueError: not enough values to unpack (expected 2, got 1)
```

Comparatively, I am able to build the R env, yet I am unable to source the `opp.R` file. Perhaps it is an issue with the ordering of instructions or the use of `here::here`?

Best,

Jeff

**Comment by jeffcsauer** [comment]
A workaround for those looking to quickly download all of the `.rds`: 

```python
import requests
from bs4 import BeautifulSoup
import re
import os
import urllib.request

# Set URL to the main page downloadsite
base_url = 'https://openpolicing.stanford.edu/data/'

# Download html
soup = BeautifulSoup(requests.get(base_url).content, "html.parser")

# Final all `a` tags, get the associated href
links = [a.get('href') for a in soup.find_all('a', href=True)]

# Get only those links where the last three characters are rds, 
# indicating an r data file
RDS_links = [link for link in links if (link[-3:] == 'rds')]

# Remove statewide files
RDS_links_cities = [link for link in RDS_links if 'statewide' not in link]

# Set wd and download
os.chdir('your_directory')

# For each link, download the file
for link in RDS_links_cities:
    urllib.request.urlretrieve(link, re.search('([^\/]+$)', link).group(0))
```

**Comment by jgaeb** [comment]
Fixed by fdb3b9f.
